### DISCUSSION TITLE
[Feature Request] MM3D Styled Clock

### DISCUSSION THREAD

**Discussion by Onemario1234** [HarbourMasters/2ship2harkinian#526] [Ideas]
The addition of a Text Only clock is actually pretty cool, but I'd love to see more settings related to the Clock. I feel like this is an obvious pick, but I thought I'd make the suggestion nonetheless. This clock, when enabled, would be a large bar at the bottom of the screen, cut into six segments, with the individual days being colorized. A small arrow would be above your current point in the cycle, with the current in-game time just above the arrow. It didn't tell if it was AM or PM, but it would be nice if the 2Ship version did. I've always liked the style of MM3D's clock, and would love to see it here.

**Comment by xsm2** [comment]
This has been implemented in 1.1, so the discussion can be closed
